css/main.css

```diff
@@ -120,6 +120,7 @@ img {
 @media screen and (min-width: 576px) {
   .main-nav {
     display: flex;
+    padding-left: 6%;
```

Comment:
You don't need this property.

Reply:
I disagree with you.
Without this property my navigation isn't like on design.
The first letter of "Services" word should has position similar to photo borders.